Add warning (#109)
ethereum/py_ecc: README.md

py_ecc
==========================
Elliptic curve crypto in python including secp256k1, alt_bn128, and bls12_381.

> **Warning: It contains some experimental codes and has NOT been audited.**

Reviewer: ```suggestion
> **Warning**: This library contains some experimental codes that have **NOT** been audited.
```
